network/topics/scoring.go

```diff
@@ -249,3 +254,17 @@ func filterCommitteesForTopic(topic string, committees []*storage.Committee) []*
 	}
 	return topicCommittees
 }
+
+// formatInvalidMessageStats returns the subnets in a small format topicNum=ti,fmd,mmd,imd
+func formatInvalidMessageStats(filtered map[string]*pubsub.TopicScoreSnapshot) string {
+	var b strings.Builder
+	i := 0
+	for topic, snapshot := range filtered {
+		if i > 0 {
+			b.WriteString(" ")
+		}
+		fmt.Fprintf(&b, "%s=%d,%0.3f,%0.3f,%0.3f", topic, snapshot.TimeInMesh, snapshot.FirstMessageDeliveries, snapshot.MeshMessageDeliveries, snapshot.InvalidMessageDeliveries)
```

Comment:
`%d` prints the number of nanoseconds for `time.Duration` which is less readable and longer than `%s`, which represents the duration in a compact format. Or is there any reason to use `%d` instead?